gemfiles/1.9.3/3.1.gemfile

```diff
@@ -0,0 +1,5 @@
+source "http://rubygems.org"
+
+gem 'rest-client', '~> 1.8.0'
+gem 'railties', '~> 3.1.0'
+gemspec :path => '../..'
```

Comment:
Same question as above, 1.9.3 is removed from our test runs, so I guess there is no sense of keeping those Gemfiles around :)

Might be a nice addition to the Compatibility section of the readme tho.